packages/client/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@near-eth/client",
-  "version": "1.3.0-3",
+  "version": "1.3.0-5",
```

Comment:
skip two versions o_O? is this because there is 1.3.0-4 in another PR?

Reply:
I did some pre-preleases for testing (not all committed) which will be deleted and the release will be made from the `Main` branch following the yarn2 release process.